requirements-test.txt

```diff
@@ -0,0 +1,6 @@
+# these are the requirements to run the full test suite.
+-r requirements.txt
+mock==3.0.5
+pytest==4.6.6
```

Comment:
See the failed travis builds:
https://travis-ci.com/kdious/purgomalum-python/builds/132778394

This requirement will ensure we can always have a version of pytest that is trusted to work.